sormas-ui/src/main/java/de/symeda/sormas/ui/importer/DocumentUploadFinishedHandler.java

```diff
@@ -67,7 +67,11 @@ public void handleFile(InputStream inputStream, String fileName, String mimeType
 							if (ok) {
 								FacadeProvider.getDocumentFacade().deleteDocument(existing);
 								try {
-									saveDocument(fileName, mimeType, length, relatedEntityType, relatedEntityUuid, bytes);
+									DocumentDto documentDto = saveDocument(fileName, mimeType, length, relatedEntityType, relatedEntityUuid, bytes);
```

Comment:
Instead of returning a dto with the fileTypeNotAllowed and fileContentAndExtensionsDoNotMatch properties, please throw a dedicated exception and handle it accordingly.

Reply:
fixed